Front-end/glaucoma-diagnosis-system/src/router/index.ts

```diff
@@ -21,12 +22,18 @@ const routes = [
         component: Register,
     },
     {
+
         path: '/reception',
         name: 'Reception',
         component: Reception,
+    },
+    {
+        path:'/adminmanage',
```

Comment:
It is better if we add "/admin" prefix for the path of all admin pages. In this case, "/admin/manage"